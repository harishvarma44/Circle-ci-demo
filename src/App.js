import React from 'react';
import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <p>
        Hello and welcome to my app
        </p>
        
        <iframe src="https://giphy.com/embed/ia7kRlpGe3IFq" title='bear-wave' data-cy="gif" width="480" height="256" frameBorder="0" className="giphy-embed" allowFullScreen></iframe>
      </header>
    </div>
  );
}

export default App;
