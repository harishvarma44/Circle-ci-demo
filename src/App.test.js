import React from 'react';
import { shallow } from 'enzyme';
import { render } from '@testing-library/react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import App from './App';

Enzyme.configure({ adapter: new Adapter() });

test('renders learn react link', () => {
  const { getByText } = render(<App />);
  const linkElement = getByText(/welcome/i);
  expect(linkElement).toBeInTheDocument();
});

it('expect to render page', () => {
  expect(shallow(<App />)).toMatchSnapshot();
})
