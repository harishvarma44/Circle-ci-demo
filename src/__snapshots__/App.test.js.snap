// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render page 1`] = `ShallowWrapper {}`;
